plugin/src/helpers/utils.ts

```diff
@@ -185,3 +187,13 @@ export const isNextAuthInstalled = (): boolean => {
     return false
   }
 }
+
+export const getCustomImageResponseHeaders = (headers: Header[]): Record<string, string> | null => {
+  const customImageResponseHeaders = headers.find((h) => h.for.startsWith('/_next/image/'))
+
+  if (customImageResponseHeaders) {
```

Comment:
```suggestion
  const customImageResponseHeaders = headers.find((header) => header.for.startsWith('/_next/image/'))

  if (customImageResponseHeaders) {
```

[dust] Prefer more verbose variables names so the code is more readable?

Reply:
```suggestion
  const customImageResponseHeaders = headers.find((h) => h.for?.startsWith('/_next/image/'))

  if (customImageResponseHeaders) {
```

I'm not sure if this could ever be missing